gui/src/components/settings/pages/GeneralSettings.tsx

```diff
@@ -116,15 +118,26 @@ const defaultValues = {
     mountingResetTaps: 3,
   },
   legTweaks: { correctionStrength: 0.3 },
-  interface: { devmode: false, watchNewDevices: true, feedbackSound: true },
+  interface: {
+    devmode: false,
+    watchNewDevices: true,
+    feedbackSound: true,
+    feedbackSoundVolume: 0.5,
```

Comment:
Just voting for a tad lower for the default, better to have it too low than too high imo. Im just assuming 0.5 is linear 50% and not logarithmic? Something like 0.2 would be preferable for myself as most of the time 50% is too high in my setup.

Reply:
I made it logarithmic, 50% will set it to ~0.25 (in whatever HTML audio units), which peaks at -16 dBFS and 20% is -24 dBFS. I believe 50% is a sane default.